ingest/ingest_pipeline.py

```diff
@@ -21,6 +21,9 @@
 # Ingest dense file
 python ingest_pipeline.py  --study-accession SCP1 --file-id 123abc ingest_expression --taxon-name 'Homo Sapiens' --taxon-common-name humans --matrix-file ../tests/data/dense_matrix_19_genes_100k_cells.txt --matrix-file-type dense
 
+# Ingest loom file
+python ingest_pipeline.py  --study-accession SCP1 --file-id 123abc ingest_expression --matrix-file ../tests/data/test_loom.loom  --matrix-file-type loom --taxon-name 'Homo Sapiens' --taxon-common-name humans
```

Comment:
```suggestion
python ingest_pipeline.py  --study-accession SCP1 --file-id 123abc ingest_expression --matrix-file ../tests/data/test_loom.loom  --matrix-file-type loom --taxon-name 'Homo sapiens' --taxon-common-name human
```

Reply:
I actually performed a git mv.... But I will make sure to do that!

Reply:
Addressed in f163441